src/utils/list.js

```diff
@@ -1,42 +1,60 @@
-const getListStyleType = (listType) => {
-  switch (listType) {
-    case 'upper-roman':
-      return 'upperRoman';
-    case 'lower-roman':
-      return 'lowerRoman';
-    case 'upper-alpha':
-    case 'upper-alpha-bracket-end':
-      return 'upperLetter';
-    case 'lower-alpha':
-    case 'lower-alpha-bracket-end':
-      return 'lowerLetter';
-    case 'decimal':
-    case 'decimal-bracket':
-    default:
-      return 'decimal';
+class ListStyleBuilder {
+  // defaults is an object passed in from constants.js / numbering with the following properties:
+  // defaultOrderedListStyleType: 'decimal' (unless otherwise specified)
+  constructor(defaults) {
+    if (!defaults) {
+      throw new Error(
+        'ListStyleBuilder requires a defaults object. See constants.js / numbering for an example.'
+      );
+    }
```

Comment:
setup default value here, like

this.defaults = defaults || { defaultOrderedListStyleType: 'decimal' }.

It doesnt feel right throwing an error.